javascript/javascript1/week2/weatherwear.js

```diff
@@ -0,0 +1,38 @@
+// 3. Weather wear
+function getClothesToWear(temperature) {
+   switch(true) {
```

Comment:
Hardcoding the true value as the condition here doesn't make the most of switch statements. It should always be a dynamic value